Content.Shared/Interfaces/GameObjects/Components/Interaction/IAttack.cs

```diff
@@ -9,18 +13,25 @@ namespace Content.Shared.Interfaces.GameObjects.Components
     /// </summary>
     public interface IAttack
     {
-        void Attack(AttackEventArgs eventArgs);
+        // Redirects to ClickAttack by default.
```

Comment:
What's more likely to happen, wide attacks or click attacks? Whichever it is should decide the redirect. I'm not sure personally, so I would say that this is fine unless you want it to be more complicated with default/virtual methods.

Reply:
Click attacks are more likely to happen imo